src/coreclr/src/debug/daccess/dacimpl.h

```diff
@@ -1706,7 +1706,11 @@ class LinearReadCache
 
         // clear the GC flag bits off the MethodTable
         // equivalent to Object::GetGCSafeMethodTable()
+#if TARGET_64BIT
+        *mt &= ~7;
```

Comment:
I assume there is no convenient central place to define this constant

Reply:
I looked but didn't find one...